packages/components/tests/dummy/app/templates/components/power-select.hbs

```diff
@@ -0,0 +1,107 @@
+{{page-title "PowerSelect Component"}}
+
+<h2 class="dummy-h2">PowerSelect</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">
+    A showcase with custom styles applied as overrides to the
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com/">ember-power-select addon</a>.
+  </p>
+
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
+
+  <h4 class="dummy-h4">States</h4>
+  <p class="dummy-text-small">Default</p>
+
+  {{! This below is just an example of invocation, to get started }}
```

Comment:
Is this comment still relevant? seems not related to the content below.

Reply:
sure not, will remove it